components/Searchbar/Searchbar.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+
+const SearchBarContainer = styled.form`
+  display: flex;
+  justify-content: center;
+  margin-bottom: 0.5rem;
+`;
+
+const SearchInput = styled.input`
+  padding: 0.3rem;
+  font-size: 0.9rem;
+`;
+
+const SearchButton = styled.button`
+  font-size: 1rem;
+`;
+
+export default function SearchBar({ setSearchTerm }) {
+  return (
+    <>
+      <SearchBarContainer id="form">
```

Comment:
If we are using a form, then the user can submit the form by pressing then enter key while it is focused. That would cause a refresh of the page (not something we want). A potential solution would be to listen to the onSubmit event and then prevent the default behaviour.

Reply:
Thanks. As the search is updated automatically, we have nothing to submit, so we adapted the structure.